lms/product/factory.py

```diff
@@ -11,13 +11,17 @@ def get_product_from_request(request) -> Product:
     from lms.product.d2l import D2L
     from lms.product.moodle import Moodle
 
-    ai = request.lti_user.application_instance
-    family = Family(request.lti_user.lti.product_family)
+    family = Family.UNKNOWN
+    settings = {}
+    if request.lti_user:
```

Comment:
I remember a comment about this while I refactored it.

When is lti_user none here? Well, now we known, in the admin pages while using the "grouping" service.

Reply:
Yes, exactly